portfolio/src/main/webapp/style.css

```diff
@@ -0,0 +1,9 @@
+::root{
+    font-family:"Roboto";
```

Comment:
use 2 spaces for indentation: https://google.github.io/styleguide/htmlcssguide.html#Indentation

Reply:
Thanks, made a change to this in commit 
61fbdbb